scipy/integrate/meson.build

```diff
@@ -0,0 +1,167 @@
+odeint_banded_test_src = ['tests/banded5x5.f']
+
+mach_lib = static_library('mach_lib', 
+    mach_src,
+    fortran_args: fortran_ignore_warnings)
+quadpack_lib = static_library('quadpack_lib', 
+    quadpack_src,
+    fortran_args: fortran_ignore_warnings)
+lsoda_lib = static_library('lsoda_lib', 
+    lsoda_src,
+    fortran_args: fortran_ignore_warnings)
+vode_lib = static_library('vode_lib', 
+    vode_src,
+    fortran_args: fortran_ignore_warnings)
+dop_lib = static_library('dop_lib', 
+    dop_src,
+    fortran_args: fortran_ignore_warnings)
+
+# TODO: Add c_args: numpy_nodepr_api
+py3.extension_module('_quadpack',
+    ['__quadpack.h', '_quadpackmodule.c'] + quadpack_src + mach_src,
+    fortran_args: fortran_ignore_warnings,
+    include_directories: [inc_np, '../_lib/src'],
+    link_with: [quadpack_lib, mach_lib],
+    dependencies: [py3_dep],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+py3.extension_module('_odepack',
+    lsoda_src + mach_src + ['_odepackmodule.c'],
+    c_args: numpy_nodepr_api,
+    fortran_args: fortran_ignore_warnings,
+    link_with: [lsoda_lib, mach_lib],
+    dependencies: [py3_dep],
+    include_directories: [inc_np],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+vode_module = custom_target('vode_module',
+  output : ['vode-f2pywrappers.f', 'vodemodule.c'],
+  input : 'vode.pyf',
+  command : [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@'],
+)
+
+py3.extension_module('vode',
+    [vode_module, fortranobject_c],
+    link_with: [vode_lib, lsoda_lib],
+    c_args: numpy_nodepr_api,
+    dependencies: [py3_dep],
+    include_directories: [inc_np, inc_f2py],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+lsoda_module = custom_target('lsoda_module',
+  output : ['lsoda-f2pywrappers.f', 'lsodamodule.c'],
+  input : 'lsoda.pyf',
+  command : [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@'],
+)
+
+py3.extension_module('lsoda',
+    [lsoda_module, fortranobject_c],
+    link_with: [lsoda_lib, mach_lib],
+    c_args: numpy_nodepr_api,
+    dependencies: [py3_dep],
+    include_directories: [inc_np, inc_f2py],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+_dop_module = custom_target('_dop_module',
+  output : ['_dop-f2pywrappers.f', '_dopmodule.c'],
+  input : 'dop.pyf',
+  command : [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@'],
+)
+
+py3.extension_module('_dop',
+    [_dop_module, fortranobject_c],
+    link_with: [dop_lib],
+    c_args: numpy_nodepr_api,
+    dependencies: [py3_dep],
+    include_directories: [inc_np, inc_f2py],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+py3.extension_module('_test_multivariate',
+    [quadpack_test_src],
+    dependencies: [py3_dep],
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
+py3.extension_module('_test_odeint_banded',
+    odeint_banded_test_src,
+    link_with: [lsoda_lib, mach_lib],
+    fortran_args: fortran_ignore_warnings,
+    install: true,
+    subdir: 'scipy/integrate'
+)
+
```

Comment:
This is still missing `py3.install_sources`.

Reply:
Ah! Yest, I forgot to add that. Thanks for pointing out.